duopull.go

```diff
@@ -392,6 +392,25 @@ func toMozLog(in interface{}) (interface{}, error) {
 	if !ok {
 		return nil, fmt.Errorf("type assertion failed on input event")
 	}
+
+	// Duo logging will store JSON data structures as strings in the log response,
+	// specifically with the event description field. Convert that into a
+	// structure here.
+	if x, ok := cv["event"]; ok {
+		y, ok := x.(map[string]interface{})
```

Comment:
NB: Any reason you are not using the inline `y, ok := x.(map[string]interface{}); ok {` here?

Reply:
Missed that one, fixed -- thanks